tests/Unit/Core/Form/IdentifiableObject/CommandBuilder/Product/SEOCommandsBuilderTest.php

```diff
@@ -155,20 +258,79 @@ public function getExpectedCommands()
             [$command],
         ];
 
-        $command = new UpdateProductSeoCommand($this->getProductId()->getValue());
-        $command->setRedirectOption(RedirectType::TYPE_CATEGORY_TEMPORARY, 51);
-        yield [
+        $command = $this
+            ->getAllShopsCommand()
+            ->setLocalizedMetaTitles($localizedMetaTitles)
+            ->setLocalizedMetaDescriptions($localizedMetaDescriptions)
+            ->setLocalizedLinkRewrites($localizedLinkRewrites)
+            ->setRedirectOption(RedirectType::TYPE_PRODUCT_TEMPORARY, 42)
+        ;
+        yield 'multi-shop only' => [
             [
                 'seo' => [
+                    'meta_title' => $localizedMetaTitles,
+                    self::MULTI_SHOP_PREFIX . 'meta_title' => true,
+                    'meta_description' => $localizedMetaDescriptions,
+                    self::MULTI_SHOP_PREFIX . 'meta_description' => 'enabled',
+                    'link_rewrite' => $localizedLinkRewrites,
+                    self::MULTI_SHOP_PREFIX . 'link_rewrite' => 1,
                     'redirect_option' => [
-                        'type' => RedirectType::TYPE_CATEGORY_TEMPORARY,
+                        'type' => RedirectType::TYPE_PRODUCT_TEMPORARY,
+                        self::MULTI_SHOP_PREFIX . 'type' => true,
```

Comment:
You should add a test case where the boolean is associated to target (to validate that any of the input can be checked and the compound will still be considered as All shops)